common/changes/@microsoft/rush/ianc-fixing-change-analyzer-perf-regeression_2018-04-12-22-20.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Fix a performance regression in the way the change analyzer is constructed.",
```

Comment:
>in the way the change analyzer is constructed. [](start = 47, length = 46)

"Fix a performance regression for \"rush build\""  might be more meaningful to the intended audience